### ISSUE TITLE
Cannot navigate content switcher with arrow keys, example is lacking

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon#1534]
### Environment
User tested with latest version of JAWS 2018, Windows 7, Chrome latest version

### Description
- [ ] Functionality and controls are not obvious to a screen reader user
- [ ] JAWS announces component as a tab control, so the user expects to navigate with arrow keys, but cannot.
- [ ] Example component is lacking adequate depiction of changing content -- which a screen reader user might use to infer functionality

### User feedback
"It is not obvious how it should behave. JAWS announces it as a tab control, but if it does not appear as such, the blind user has no other way to tell you that it is something different. Thus, one would expect to navigate this using the arrow keys which does not work. Also, there is no sample content that would change. Confusing to the user."

**Comment by dakahn** [comment]
Will need to circle back around on this post v10 and confirm this is still an issue.
